.prettierrc

```diff
@@ -12,9 +12,15 @@
     "files": [
       "**/{codemods,eslint,packages}/*/{src,test}/**/*.{js,ts}"
     ],
+    "excludeFiles": ["**/packages/babel-helpers/src/helpers/**/*.js"],
     "options": {
       "trailingComma": "all"
     }
+  }, {
+    "files": "**/packages/babel-helpers/src/helpers/**/*.js",
+    "options": {
+      "trailingComma": "es5"
+    }
   }, {
```

Comment:
This seems to be working when using prettier directly, but when using `eslint-plugin-prettier` it still reformats the file adding trailing commas :thinking:

Reply:
Isn't it adding the trailing comma only to objects and arrays?

Reply:
Yes, but for helpers we don't want it since many helpers are es5-compatible.

Reply:
Oh thanks, I see it now. They were supported in es5 :upside_down_face:

Reply:
On the prettier website they state that the trailing comma in objects and arrays are valid in es5?

Reply:
Yes I just was 100% sure that they were added in es6, I was wrong. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas